### PR TITLE
fix(config): correct ZWA012 parameters indexing

### PR DESCRIPTION
**Pull request by ScratMan**
Correct the configuration parameters indexes and add missing based on updated engineering sheets from Aeotec.

### CODE REVIEW

**Comment by ScratMan** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
@blhoward2

**Comment by blhoward2** [comment]
I have no idea what happened here. Please take a look at the style guide in the project docs. The options here don't conform.

**Comment by ScratMan** [comment]
The original config files of ZWA011 and ZWA012 were based on outdated engineering sheets that didn't match the user manual and the final product released to the market. I created a ticket at Aeotec, and they confirmed the issue. They finally released new versions of the engineering sheets. The changes are not linked to a firmware update (firmware is in v1.0).

I shortened the labels of the parameters and options in the config file.

File: packages/config/config/devices/0x0371/zwa012.json [hunk 1]
```diff
@@ -47,52 +63,142 @@
 			"$import": "~/0x0086/templates/aeotec_template.json#sensor_operation_mode"
 		},
 		"2": {
-			"$import": "~/0x0086/templates/aeotec_template.json#invert_state_report"
+			"$import": "~/0x0086/templates/aeotec_template.json#external_switch_3way",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa012.json]: There's not really a point in doing that import when you're changing everything but the `minValue`. Just write this parameter normally if there is no good template for it.

File: packages/config/config/devices/0x0371/zwa012.json [hunk 2]
```diff
@@ -47,52 +63,142 @@
...
+				},
+				{
+					"label": "Momentary mode inverted",
+					"value": 3
+				}
+			]
 		},
-		"3[0x02]": {
+		"3": {
+			"$import": "~/0x0086/templates/aeotec_template.json#invert_state_report",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa012.json]:
```suggestion
			"$import": "~/templates/master_template.json#base_0-1_nounit",
```
Don't import a second-level template if you're overwriting everything it changes from the original template.

File: packages/config/config/devices/0x0371/zwa012.json [hunk 3]
```diff
@@ -47,52 +63,142 @@
...
 		},
 		"14": {
-			"$import": "~/0x0086/templates/aeotec_template.json#tilt_sensor"
+			"$import": "~/0x0086/templates/aeotec_template.json#central_scene_functionality"
 		},
 		"15": {
+			"$import": "~/0x0086/templates/aeotec_template.json#tilt_sensor"
+		},
+		"16": {
+			"$import": "~/0x0086/templates/aeotec_template.json#invert_state_report",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa012.json]:
```suggestion
			"$import": "~/templates/master_template.json#base_0-1_nounit",
```
dito

File: packages/config/config/devices/0x0371/zwa012.json [hunk 4]
```diff
@@ -47,52 +63,142 @@
...
+		},
+		"19": {
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_basic_set_on",
+			"label": "Association Group 5: Basic Set Value (On)"
+		},
+		"20": {
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_basic_set_off",
+			"label": "Association Group 5: Basic Set Value (Off)",
+			"defaultValue": 0
+		},
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa012.json:182]: @blhoward2 what do you think? I'm not sure if importing from a "group 2" template and then changing everything to a 5 is better than just importing the original `"~/templates/master_template.json#base_0-255_nounit"`

File: packages/config/config/devices/0x0371/zwa012.json [hunk 5]
```diff
@@ -47,52 +63,142 @@
...
+		"23": {
+			"$import": "~/0x0086/templates/aeotec_template.json#low_battery_threshold",
+			"defaultValue": 20
+		},
+		"24": {
+			"$import": "~/templates/master_template.json#base_0-255_nounit",
+			"label": "Battery report interval",
+			"defaultValue": 24,
+			"unit": "hour",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0371/zwa012.json]:
```suggestion
			"unit": "hours",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
@ScratMan FYI it might take a few days until @blhoward2 can respond.

File: packages/config/config/devices/0x0371/zwa012.json (hunk 4 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0371/zwa012.json:182]: I'm fine either way. Where the ones for other groups get repeated over devices I've also templated those in the past just so it's easier to rename them all should we ever want to do so.

Comment (AlCalzone) [reply packages/config/config/devices/0x0371/zwa012.json:182]: okay, we can keep these then.

File: packages/config/config/devices/0x0371/zwa012.json (hunk 1 above)
Comment (ScratMan) [reply packages/config/config/devices/0x0371/zwa012.json]: OK, done

**Review by AlCalzone — DISMISSED** [review]

**Comment by AlCalzone** [comment]
LGTM - I'll let @blhoward2 take another look as he was working on all the Aeotec files before.

**Review by blhoward2 — APPROVED** [review]

**Pull request merged**